layers/+lang/c-c++/packages.el

```diff
@@ -23,7 +23,7 @@
     helm-cscope
     helm-gtags
     semantic
-    (srefactor :toggle (version< emacs-version "24.4"))
+    (srefactor :toggle (not (version< emacs-version "24.4")))
```

Comment:
@JAremko I find the keyword `:toggle` a bit odd. Maybe it can just take a conditional form like `(if ...` or `(unless ...` etc if I'm understanding its purpose correctly.